### PR TITLE
Disconnect port aft solar control's SMES input from the main power line on MetaStation

### PR DESCRIPTION
**Pull request by iprice**
## About The Pull Request
Relocates and rewires the SMES on Port Aft (south west) solar control on MetaStation
This SMES is currently wired directly into the main power lines (check out its power input, it's drawing from the engine room SMES by default) and also connects the solars straight into the main power line.
Relocated the SMES, generated new wire terminal directional, SMES is now fed only by the local solar array and outputs into the main grid.
Local APC still fed by main power line.  Tested total supply on the lines at various points before and after wiring the array.

This is my first map change so please give it a check; the final diff looks reasonable though.  Particularly I don't understand layering, but the map compiles and plays as I'd expect in the altered area.

## Why It's Good For The Game
Consistency, perhaps some deeper reasoning behind the power grid.  The other 3 solar controls are wired up to run solars->SMES-> main grid, but this one room basically just wires them all together directly.
Prevents engineers from asking questions about the wiring of this room :D

## Changelog
:cl: iain0
fix: MetaStation Port Aft Solar Control SMES input isolated from main power line, like all other solar control rooms.
/:cl:

### CODE REVIEW

**Comment by 81Denton** [comment]
>Consistency, perhaps some deeper reasoning behind the power grid

By default, solars/grav gen/AI sat are disconnected from the main power grid to make power sink sabotage harder. Not that people use power sinks much.

**Review by Cyberboss — APPROVED** [review]

**Comment by Anonmare** [comment]
@81Denton He means, the solars he's correcting are broken and he's making them consistent with the rest of them.

In plain english, the battery is plugged in backwards.

**Comment by BadSS13Player** [comment]
IIRC DeltaStation has the same problem on 2 places.
